docs/html/ux-research-design/research-results/users-and-security.md

```diff
@@ -76,8 +76,9 @@ Participants also explained they rely on code security scanning and checking sof
 
 #### Reliance on good software development practices
 
-A small number of participants e### Selected quotes from research participants
-xplained they have good software practices in place, which help with writing secure software.
+Selected quotes from research participants
+
+> A small number of participants explained they have good software practices in place, which help with writing secure software.
```

Comment:
Any reason to believe that this was a quote? It seems more likely that this was the preamble to the section header that moved in the middle of the paragraph for some reason...